lib/osf-components/addon/components/debouncer/component.ts

```diff
@@ -0,0 +1,35 @@
+import Component from '@ember/component';
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+import { timeout } from 'ember-concurrency';
+import { task } from 'ember-concurrency-decorators';
+
+import { layout } from 'ember-osf-web/decorators/component';
+
+import template from './template';
+
+@layout(template)
+export default class Debouncer extends Component {
+    fn!: () => void;
+    timeoutInterval!: number;
```

Comment:
Should probably have a default value.

Reply:
I feel like for a debouncer the user must specify a timeout value instead of relying on defaults.

Reply:
I agree. IMO, you can specify an override value if need be for your case but in most cases there is no need to emphasize on the timeout value in the component invocation.